pkg/config/orderer_test.go

```diff
@@ -221,65 +232,453 @@ func TestUpdateOrdererConfiguration(t *testing.T) {
 	updatedOrdererConf.BatchSize.MaxMessageCount = 10000
 	updatedOrdererConf.Addresses = []string{"newhost:345"}
 	updatedOrdererConf.OrdererType = ConsensusTypeEtcdRaft
+	updatedOrdererConf.EtcdRaft = EtcdRaft{
+		Consenters: []Consenter{
+			{
+				Host:          "host1",
+				Port:          123,
+				ClientTLSCert: &x509.Certificate{},
+				ServerTLSCert: &x509.Certificate{},
+			},
+		},
+		Options: EtcdRaftOptions{},
+	}
 
 	err = UpdateOrdererConfiguration(config, updatedOrdererConf)
 	gt.Expect(err).NotTo(HaveOccurred())
 
-	// Expected OrdererValues
-	expectedCapabilities, err := proto.Marshal(&cb.Capabilities{
-		Capabilities: map[string]*cb.Capability{
-			"V1_3": {},
+	expectedConfigJSON := `
+{
+	"channel_group": {
+		"groups": {
+			"Orderer": {
+				"groups": {
+					"OrdererOrg": {
+						"groups": {},
+						"mod_policy": "Admins",
+						"policies": {
+							"Admins": {
+							"mod_policy": "Admins",
```

Comment:
Seems we have whitespace stuff going on again. Not sure why the indentation keeps getting lost.

Reply:
it might be because when I copy the output from my terminal the text wrapping is removing tab characters? feels unlikely but I'll keep an eye out again for when it happens. I'll leave as is for now until we have to change this expected when we update something that touches it again